scripts/update_protos.sh

```diff
@@ -35,3 +37,9 @@ curl ${POOL_URL}/${POOL_RELEASE_TAG}/auctioneerrpc/auctioneer.proto --create-dir
 curl ${POOL_URL}/${POOL_RELEASE_TAG}/auctioneerrpc/hashmail.proto --create-dirs -o protos/pool/${POOL_RELEASE_TAG}/auctioneerrpc/hashmail.proto
 
 curl ${FARADAY_URL}/${FARADAY_RELEASE_TAG}/frdrpc/faraday.proto --create-dirs -o protos/faraday/${FARADAY_RELEASE_TAG}/faraday.proto
+
+# TODO: uncomment these lines once the autopilot protos have been merged into the public LiT repo.
```

Comment:
this is dependent on https://github.com/lightninglabs/lightning-terminal/pull/487?

Reply:
yup. i'll need to update this PR after that gets merged and shipped in the next release.